controller/monitor/environment_check_monitor.go

```diff
@@ -190,10 +188,10 @@ func (m *EnvironmentCheckMonitor) syncPackagesInstalled(kubeNode *corev1.Node, n
 	case strings.Contains(osImage, "ubuntu"):
 		fallthrough
 	case strings.Contains(osImage, "debian"):
-		queryPackagesCmd = "dpkg"
-		options = append(options, "-l")
-		packages = append(packages, "nfs-common", "open-iscsi", "cryptsetup", "dmsetup")
-		pipeFlag = true
+		packageProbeExecutables["nfs-common"] = "mount.nfs"
```

Comment:
We don't directly use mount.nfs in Longhorn. Just ensure the system NFS capability is fulfilled.